### ISSUE TITLE
Unable to compile nannou_audio on OS X

### ISSUE THREAD

**Issue by julesyoungberg** [nannou-org/nannou#704]
Hi, I'm really stoked on this project and can't wait to use it more. I would like to use nannou_audio but am unable to compile it. I can't run any of the audio examples and I can't add nannou_audio to my own project, I get the same error in both cases. 

OS X version: 10.15.7
Rust version: 1.49.0
Nannou version: 0.15

Log output:
```
   Compiling sample v0.10.0
   Compiling coreaudio-sys v0.2.8
error[E0282]: type annotations needed
  --> /Users/jules/.cargo/registry/src/github.com-1ecc6299db9ec823/sample-0.10.0/src/window.rs:73:41
   |
73 |         let v = phase.to_float_sample().to_sample() * PI_2;
   |                                         ^^^^^^^^^ cannot infer type for type parameter `S` declared on the associated function `to_sample`
   |
help: consider specifying the type argument in the method call
   |
73 |         let v = phase.to_float_sample().to_sample::<S>() * PI_2;
   |                                                  ^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0282`.
error: could not compile `sample`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

I think I'll just use cpal directly for now but it would be awesome to be get this working.

**Comment by zmitchell** [comment]
This looks like the same issue as #703.

**Comment by julesyoungberg** [comment]
Yep, my bad, didn't read that because it had Windows in the title.

**Comment by mgrider** [comment]
I wanted to play with nannou also, but ran into this issue. I tried the recommended fix of changing `Cargo.toml` to specify audrey 3.0, but that didn't fix it for me.

I don't know what it means to "use cpal directly".

**Comment by julesyoungberg** [comment]
nannou_audio is built on the cpal crate, you can use it for audio io. Here's the docs: https://docs.rs/cpal/0.13.1/cpal/

**Comment by cpdean** [comment]
tracked it down to this dependency: https://github.com/RustAudio/dasp/issues/141

dependency is wired up here https://github.com/nannou-org/nannou/blob/2cedce1014053c9771e592f8841d24006a05d9b2/nannou_audio/Cargo.toml#L15

while the master branch of that repo builds, the maintainer has not cut a release. 0.11.0 was [released on may 29, 2020](https://crates.io/crates/dasp_sample/0.11.0), i think that crate has the last git sha of may 29, 2020. reproducing it here:

```
git clone git@github.com:RustAudio/dasp.git
cd dasp
git checkout 16f7498dcaae4201219bd6fecad94724686de2e6 
cargo build
```

**Comment by mitchmindtree** [comment]
Thanks for the report! Closed via #665.
